custom_components/tuya_local/devices/ultenic_air_fryer.yaml

```diff
@@ -122,19 +121,18 @@ secondary_entities:
         type: string
         name: unit
         mapping:
-          - dps_val: "0"
+          - dps_val: c
             value: C
-          - dps_val: "1"
+          - dps_val: f
             value: F
   - entity: sensor
-    name: Temperature
+    name: Internal Temperature
```

Comment:
HA naming convention is that only the first word in a name gets capitalised, so if changing this the t also needs changing.